docs/how-to/add-an-application.mdx

```diff
@@ -76,70 +95,65 @@ Select the [authenticator configuration type](/docs/next/authentication).
     </TabItem>
   </Tabs>
 
-
-
-Click **Submit**. You'll be taken to the app page for the new application. Here, you'll be able to a number of things, such as edit the basic and advanced settings or configure the signup/login experience. 
-
+Click **Submit**. You'll be taken to the app page for the new application. Here, you'll be able to a number of things, such as edit the basic and advanced settings or configure the signup/login experience.
 
 ## API
 
-Before creating an application by API, you must have an [API access token](/docs/next/create-api-token#api). You'll also need to create a Resource Server and an Authenticator Config, which you'll do below. A Resource Server is a namespace for application scopes that are a set of all scopes supported by the application. An Authenticator Config defines the invocation type, invoke URL, and trusted origins. 
+Before creating an application by API, you must have an [API access token](/docs/next/create-api-token#api). You'll also need to create a Resource Server and an Authenticator Config, which you'll do below. A Resource Server is a namespace for application scopes that are a set of all scopes supported by the application. An Authenticator Config defines the invocation type, invoke URL, and trusted origins.
 
-1. Create a [Resource Server](/docs/next/add-resource-server#api). A Resource Server is a namespace for application scopes that are a set of all scopes supported by the application. 
+1. Create a [Resource Server](/docs/next/add-resource-server#api). A Resource Server is a namespace for application scopes that are a set of all scopes supported by the application.
```

Comment:
For all of these numbered items, I think we want to maintain the indentations:

| Before | After |
| --- | --- |
| <img width="1728" alt="Screenshot 2023-09-25 at 3 00 51 PM" src="https://github.com/gobeyondidentity/developer-docs/assets/4763133/e35b8af6-eb18-4958-bb1f-a3d1aec6b545"> | <img width="1728" alt="Screenshot 2023-09-25 at 3 01 00 PM" src="https://github.com/gobeyondidentity/developer-docs/assets/4763133/84c2fe69-c120-481e-a47a-279eda78f347"> |

Reply:
good catch, updated. It's hard to prevent prettier in vscode from removing these indentations.